openedx/openedx-platform
# feat: Discussion Topics API v2 [BD-38] [TNL-9332] [BB-5189]

## Description

Adds a new discussion topic API that uses new discussion topic links. Supports the new provider, support sorting by name, activity or course structure. 

## Testing instructions

- Switch a course to using the new discussions provider. 
- Check out:  http://localhost:18000/api/discussion/v2/course_topics/{course_id} 
- It should return the topics in the course for each unit based on the settings. 
-  You should be able to filter by topic id and sort by activity, strucure, or name
